src/main/java/de/theit/jenkins/crowd/CrowdConfigurationService.java

```diff
@@ -247,9 +267,12 @@ public boolean isGroupMember(String username) {
         // Load the entry from cache if it's valid return it
         Boolean retval = getValidValueFromCache(username, isGroupMemberCache);
         if (retval != null) {
+            LOG.log(Level.FINEST, "isGroupMember() cache hit: {0}", username);
             return retval;
         }
 
+        LOG.log(Level.FINEST, "isGroupMember() cache hit MISS: {0}", username);
+
```

Comment:
Please change this log to debug. Mostly because it could be pretty extensive in bigger organization.